### PR TITLE
Update test script to run partly in parallel

### PR DESCRIPTION
**Pull request by domoscargin**
#209 is failing because tests are timing out (some investigation here: https://github.com/alphagov/digitalmarketplace-govuk-frontend/pull/211).

This is an attempt to fix things by running `standard` and the compile for Jest in parallel.

It's not immediately clear how to robustly test this solution, however.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,7 +8,7 @@
   "scripts": {
     "dev": "gulp dev --gulpfile 'app/gulpfile.js'",
     "heroku": "node app/start.js",
-    "test": "standard && gulp compile && jest",
+    "test": "standard & gulp compile && jest",
```
Comment (bjgill) [inline package.json:11]: Is this a shell script? If so, what happens if `standard` fails? It looks as if failures might be missed...

**Comment by bjgill** [comment]
Just ran this branch and still getting failures of the same type:

>     : Timeout - Async callback was not invoked within the 5000 ms timeout specified by jest.setTimeout.Timeout - Async callback was not invoked within the 5000 ms timeout specified by jest.setTimeout.Error:

**Comment by domoscargin** [comment]
I want to eliminate the possibility of weirdness with standard as the cause - what happens if you simply remove standard from the test script?

https://github.com/alphagov/digitalmarketplace-govuk-frontend/blob/0110acc2cdc33d9ed525145940e5e1b339866eef/package.json#L11

**Comment by domoscargin** [comment]
I'm going to close this and continue to mess around on my WIP branch.

**Comment by lfdebrux** [comment]
> Just ran this branch and still getting failures of the same type:
> 
> > ```
> > : Timeout - Async callback was not invoked within the 5000 ms timeout specified by jest.setTimeout.Timeout - Async callback was not invoked within the 5000 ms timeout specified by jest.setTimeout.Error:
> > ```
> 
> EDIT: I seem to be able to repro this fairly reliably on my machine. Give me a shout if you need a guinea pig for your testing

Maybe it's a Linux only issue

**Pull request closed without merging**